### PR TITLE
Add a machine to xenobio so they can actually contribute and not be worthless

### PR DESCRIPTION
**Pull request by MajManatee**
# Document the changes in your pull request

Adds a plort refinery machine to xenobio that turns slime cores into research points! Finally something that makes the "Research" you are doing not worthless!

# Spriting
yeah there probably will be some sprites when I get around to it, or if someone else makes them idk it would be nice :)

# Wiki Documentation
will probably have to be added to the xenobio section or something
# Changelog
:cl:  
rscadd: Added plort refinery
wip: This is work in progress  
mapping: put the thingy on the maps where it needs to be.
/:cl:

### CODE REVIEW

**Comment by ToasterBiome** [comment]
research values should probably be on the slime core instead of the machine, nice idea though
will need a cap, either total slime cores or x of each core, maybe diminishing returns

**Comment by Cartlord** [comment]
I like the idea, though if it's just a matter of numbers, it might end up being optimal to just make a fuckton of greys - perhaps add diminishing returns to some of the lower-tier extracts, or a special effect for high-tier ones?

**Comment by MajManatee** [comment]
you over-estimate my abilities with the special effect thing
putting it in the core is probably a good idea, if I can figure that out

File: code/modules/research/plortmachine.dm [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/**********************Plort Redemption Machine**************************/
+//Accepts slime "Cores" as they are definetly NOT called, and turns them into stuff. Probably money? maybe goods? idk im writing this thing before i code it.
+
+/obj/machinery/plortrefinery
+	name = "plort redemption machine"
```
Comment (drderp3635) [inline code/modules/research/plortmachine.dm:5]: plorts are from slime rancher, it should be called an extract redemption machine to match our vernacular

Comment (VaelophisNyx) [reply code/modules/research/plortmachine.dm:5]: more slimerancher = good

Comment (ReddicusDragon) [reply code/modules/research/plortmachine.dm:5]: I am in agreement with drderp

**Review by ReddicusDragon — CHANGES_REQUESTED** [review]
slime rancher bad

**Comment by Bibby0110** [comment]
Stale for many days, lovely idea, please make it happen :)

**Pull request closed without merging**